install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -113,6 +113,79 @@ value, triggers provisioning using the single implied
 link:#enabling-provisioner-plugins[provisioner plug-in per cloud].
 ====
 
+
+[[storage-provisioner-labels]]
+=== Storage-Provisioner Labels
+
+For dynamically provisioned storage, OpenShift defines three key/value pairs,
+called _storage-provisioner labels_ in this document,
+and associates them with the storage, in a storage-specific way.
```

Comment:
"...and associates them with the storage, in a storage-specific way."

This last bit (storage-specific) sounds a bit cryptic to me. Could you expand on it or explain it more? It also feels like a run-on sentence. Perhaps:

For dynamically provisioned storage, OpenShift defines three key/value pairs, called _storage-provisioner labels_ in this document. These labels are associated with storage, in a storage-specific way..."

^ except instead of saying "in a storage-specific way," you could explain what that actually means. Maybe I'm missing something, but "in a storage-specific way" does not add any meaning, to me.